test/tests/gcc-c-hello-world/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+image="$1"
+dirTest="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+dirContainer='/usr/src/c'
+
+docker run --rm -v "$dirTest":"$dirContainer":rw -w "$dirContainer" "$image" gcc -o hello-world  hello-world.c
+"$dirTest"/hello-world
+rm -rf "$dirTest"/hello-world
```

Comment:
@MBonell Did you use exactly the command that @yosifkit suggested? It should be writing `hello-world` to `/`, which should be writable. It seems like you may have done `sh -c 'gcc -o hello-world hello-world.c && hello-world'` instead.